lib/unique_elements.rb

```diff
@@ -1,6 +1,21 @@
 # This method takes an array of elements (these might be strings, integers, floats, or a
 # combination of the above), and returns an array of the elements that appear once and only
 # once.
-def find_unique_elements(arr)
+
+def count_elements(arr)
```

Comment:
This method does its operations correctly, but it doesn't have a valid **return value** at the end. A method's last line should be `return ________` – as a way of explicitly defining what the final value that we care about is.

In this case, you'll want to return the hash that is collecting all the values and their respective frequencies.